types/constants.go

```diff
@@ -49,6 +49,9 @@ var (
 	SiafundCount     = NewCurrency64(10000)
 	SiafundPortion   = big.NewRat(39, 1000)
 	TargetWindow     BlockHeight
+
+	// size of each page for paginated API responses
+	DefaultPaginationSize = 50
```

Comment:
We usually try to keep the constants in a `consts.go` file within the module it is used in. Since it is only 1 declaration you can probably add `defaultPaginationSize` to a `node/api/pagination.go` file together with the code or move it into a `node/api/consts.go` file.